source/shared/net/concepts/id-accounts.rst

```diff
@@ -1,115 +1,86 @@
-
-Benefits
-========
-
-Every account on the chain must be derived from an identity that is verified and
-signed by an approved identity provider. It is publicly visible which identity
-provider issued an identity for an account, and who the anonymity revokers are
-for the account and the identity. In addition to this basic information which
-enables regulatory compliance, an account owner can choose to publicly reveal
-other values on their account, such as their nationality or country of
-residence. Since this information is publicly accessible anybody can check it
-before interacting with an account. Moreover, being able to see who issued the
-identity enables whoever wishes to interact with an account to judge the level
-of risk in the transaction.
-
-.. _revoking-anomity:
-
-Revoking anonymity
-==================
-
-When necessary, the anonymity revokers and identity provider can work together
-to determine the owner of an account and determine which accounts belong to the
-same owner. (They should only do so when legally obliged to, such as by a court
-order.) Anonymity revocation is a two-stage process requiring cooperation of
-multiple parties.
-
-1. Each account has an encryption of a specific *user identifier*. This
-   number can be decrypted by sufficiently many of the anonymity revokers
-   working together. (The set of anonymity revokers and the number of them
-   required to decrypt the user identifier are determined when the identity is
-   issued.)
-2. Once the user identifier is decrypted the identity provider can look
-   up the real-life identity of the owner of the account.
-
-After step 2 the anonymity revokers can additionally decrypt a value that is
-held by the identity provider and allows the revokers to find all accounts the
-user has created from a given identity. Additionally, this value allows
-anonymity revokers to see the amount of CCDs in the shielded balance of
-de-anonymized accounts.
-
-All of these actions are subject to rules and processes, and only the relevant
-entities learn any information about the account owner. No information is
-publicly revealed.
+Identity issuance requires *Identity Verification*, which is the process of verifying the real-life identity of the user. This typically requires taking photographs or scans of identification documents, such as a passport. Identity verification also checks that the user-chosen attributes are valid for the user.
+
+Upon verification of the user's identification documents and attributes, the Identity provider issues a :ref:`user identity certificate<glossary-user-identity-certificate>`. The User identity certificate contains attributes about the user. It is basically the Identity Provider’s signature over some cryptographic keys of the user and the validated personal attributes.
+
+.. image:: ../images/concepts/identity-creation.png
+   :alt: graphic drawing showing how the user interacts with the identity provider
+
+About accounts
```

Comment:
This section seems to be almost identical to what is on the Accounts page. Is this intended?

Reply:
The two drawings should be different. One should show purely identity flow and one should show how the user can create accounts.